packages/styling/src/styles/DefaultFontStyles.ts

```diff
@@ -1,66 +1,55 @@
-import { fontFace, CSSProperties } from 'glamor';
+import { IFontStyles, IRawStyle } from '../interfaces/index';
+import { fontFace } from '../glamorExports';
 
 const FONT_BASE_URL: string = 'https://static2.sharepointonline.com/files/fabric/assets/fonts/segoeui-westeuropean';
 const ICON_BASE_URL: string = 'https://static2.sharepointonline.com/files/fabric/assets/icons';
 
 const SYSTEM_BASE: string = '"Segoe UI", -apple-system, BlinkMacSystemFont, "Roboto", "Helvetica Neue", sans-serif';
 const FAMILY_BASE: string = '"Segoe UI WestEuropean", ' + SYSTEM_BASE;
 
-const LIGHT_WEIGHT: number = 100;
-const SEMILIGHT_WEIGHT: number = 300;
-const REGULAR_WEIGHT: number = 400;
-const SEMIBOLD_WEIGHT: number = 600;
-
-const MINI_SIZE: string = '10px';
-const X_SMALL_SIZE: string = '11px';
-const SMALL_SIZE: string = '12px';
-const SMALL_PLUS_SIZE: string = '13px';
-const MEDIUM_SIZE: string = '14px';
-const MEDIUM_PLUS_SIZE: string = '15px';
-const LARGE_SIZE: string = '17px';
-const X_LARGE_SIZE: string = '21px';
-const XX_LARGE_SIZE: string = '28px';
-const SUPER_SIZE: string = '42px';
-const MEGA_SIZE: string = '72px';
+export namespace FontSizes {
```

Comment:
What's the reason behind the namespace export instead just exporting an object with these properties?